lib/rubocop/cop/rspec/named_subject.rb

```diff
@@ -0,0 +1,68 @@
+# encoding: utf-8
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Avoid referencing `subject` explicitly in specs
```

Comment:
Hm well the other messages don't explain why the code is bad but many do provide links. Maybe I'll try to provide a link in the YARD docs